src/components/Header/SideNav/index.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import styled from 'styled-components'
+import remcalc from 'remcalc'
+import Flex from 'styled-flex-component'
+import is from 'styled-is'
+
+import CloseButton from './CloseButton'
+import Dropdown from './Dropdown'
+import OuterAnchorItem from './OuterAnchorItem'
+
+const SideNavList = styled.ul``
+
+const SideNavPanel = styled.div.attrs({
+  as: 'nav'
+})`
+  position: fixed;
+  background: ${props => props.theme.colors.blueBg};
+  height: 100vh;
+  width: 100vw;
+  left: 0;
+  top: 0;
+  z-index: ${props => props.theme.zIndexes.header};
+  transform: translateX(100%);
+  transition: transform ${props => props.theme.animations.fast} ease-in-out;
+
+  /* Thinner version' of the navbar */
+  @media screen and (min-width: 600px) and (max-width: 959px) {
+    width: ${remcalc(295)};
+    left: auto;
+    right: 0;
+    z-index: 10;
+  }
+
+  ${is('open')`
+    transform: translateX(0%);
+  `};
+
+  @media screen and (min-width: 960px) {
+    display: none;
+  }
+`
+
+const SideNav = ({ links, isOpen, onClose }) => (
+  <SideNavPanel open={isOpen}>
+    <Flex justifyEnd>
+      <CloseButton onClick={onClose} />
+    </Flex>
+    <SideNavList>
+      <OuterAnchorItem activeClassName="active" to="/">
+        Home
+      </OuterAnchorItem>
+      {links.map((link, idx) => {
+        if (link.dropdownItems) {
```

Comment:
I'd join lines 53-71 into a `<SideNavItem item={link}>` that does the logic to check for dropdownItems inside

Reply:
Not sure if this is quite what you envisaged?